src/main/java/org/moltimate/moltimatebackend/service/ActiveSiteService.java

```diff
@@ -22,7 +21,6 @@
  * active site table.
  */
 @Service
-@Validated
```

Comment:
What does this do?

Reply:
It's an annotation that lets entities/embeddables have certain properties be validated based on constraints. I don't know how it made its way into a service, it shouldn't have been there.

Reply:
👍